cluster/gce/util.sh

```diff
@@ -3342,6 +3364,8 @@ function create-rpmaster {
   MASTER_ADVERTISE_ADDRESS="${RPSERVER_RESERVED_INTERNAL_IP[$rp_sequence]}"
   KUBERNETES_MASTER_INTERNAL_IP="${RPSERVER_RESERVED_INTERNAL_IP[$rp_sequence]}"
   KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig${KUBEMARK_PREFIX}.rp-$rp_sequence"
+  MIZAR_VPC_RANGE_START=""
```

Comment:
do these setting empty really needed?

Reply:
Not sure what impact if we give RP KCM vpc_range. so leave it empty, then will not pass to RP KCM. otherwise, these will be always the last TP's value.

Reply:
> Not sure what impact if we give RP KCM vpc_range. so leave it empty, then will not pass to RP KCM. otherwise, these will be always the last TP's value.

Mizar controller won't be started in RP KCM. Argument will be taken but useless in RP.